README.md

```diff
@@ -103,32 +113,142 @@ and deploying.
 
 Hot-fixes are emergency fixes made to the current release that bypass changes
 that are in `main`. These fixes should still go through the pull-request
-process, but to a version-specific branch, e.g. `v1.1.1-hotfixes`. Once these
+process, but to a version-specific branch, e.g. `v1.0.1-hotfixes`. Once these
 are merged in, the commits will need to be applied to `main`, e.g. via
 cherry-picking, and `release` should be fast-forwarded/reset to the latest code
 released.
 
+```mermaid
+gitGraph
+    branch release
+    commit tag: "v0.9.9"
+    checkout main
+    commit tag: "v1.0.0-rc1, v1.0.0"
+    checkout release
+    merge main tag: "v1.0.0"
+    checkout main
+    commit id: "v1.1.0-feature-1"
+    commit id: "v1.1.0-feature-2"
+    checkout release
+    branch v1.0.1-hotfixes
+    commit id: "v1.0.1-hotfix-1"
+    checkout release
+    merge v1.0.1-hotfixes tag: "v1.0.1"
```

Comment:
Would it be possible to show how `hotfix-1` gets into `main` too? Not sure if there's a good way to show this with Mermaid's git graphs.